lib/puzzles/widgets/puzzle_types/puzzle_single_choice.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_and_friends/puzzles/cubit/puzzles_cubit.dart';
+import 'package:flutter_and_friends/puzzles/helper/puzzle_helper.dart';
+import 'package:flutter_and_friends/puzzles/widgets/fox_animation.dart';
+import 'package:flutter_and_friends/puzzles/widgets/widgets.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:puzzle_repository/puzzle_repository.dart';
+
+class PuzzleSingleChoice extends StatelessWidget {
+  const PuzzleSingleChoice({required this.puzzle, super.key});
+
+  final Puzzle puzzle;
+
+  @override
+  Widget build(BuildContext context) {
+    final foxController = FoxController(audioType: AudioType.singleChoice);
+    String? selectedAnswer;
```

Comment:
This is state that's being maintained in the `build` method which is generally not something you want to do since this widget can rebuilt at any time and lose it's `selectedAnswer`. This should be refactored into a `StatefulWidget` instead if you want to maintain the `selectedAnswer` as state.